github/SelfHostedActionsRunner.pyi

```diff
@@ -1,19 +1,19 @@
-from typing import Any, Dict, List, Union
-
-from github.GithubObject import NonCompletableGithubObject
-
-class SelfHostedActionsRunner(NonCompletableGithubObject):
-    def __repr__(self) -> str: ...
-    def _initAttributes(self) -> None: ...
-    def _useAttributes(self, attributes: Dict[str, Any]) -> None: ...
-    @property
-    def id(self) -> int: ...
-    @property
-    def name(self) -> str: ...
-    @property
-    def os(self) -> str: ...
-    @property
-    def status(self) -> str: ...
-    @property
-    def busy(self) -> bool: ...
-    def labels(self) -> List[Dict[str, Union[str, int]]]: ...
+from typing import Any, Dict, List, Union
```

Comment:
This looks unrelated?

Reply:
Missed that one, sorry. We added a bunch of stuff for self-hosted runners in our fork.

Reply:
Actually, this is a change caused by wrong line endings. By loading the file, my IDE changed it from `\r\n` to `\n`. Since the latter is used all over the repo, I think this is a good change. You can see there is no actual change by checking `Hide whitespaces` in the cog wheel menu at the top of the PR.